plone/app/upgrade/v61/configure.zcml

```diff
@@ -0,0 +1,19 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:gs="http://namespaces.zope.org/genericsetup"
+    i18n_domain="plone"
+    >
+
+  <gs:upgradeSteps
+      profile="Products.CMFPlone:plone"
+      source="6099"
```

Comment:
Does this work even though 6099 isn't the current version from the CMFPlone 6.0 branch?

Reply:
Yes.  We do the same for [v60](https://github.com/plone/plone.app.upgrade/blob/37b4d7dc7ff48e15a2656e2fea71fede036fd84f/plone/app/upgrade/v60/configure.zcml#L9-L12) and [v52](https://github.com/plone/plone.app.upgrade/blob/37b4d7dc7ff48e15a2656e2fea71fede036fd84f/plone/app/upgrade/v52/configure.zcml#L9-L12).  GenericSetup is smart enough to use this to get a correct order of execution, even when a source does not exist.  I expect that `source="*"` would work too, but I don't want to be creative here. :-)